### DISCUSSION TITLE
Setting dark mode (dynamic) seems to do the job all again when I close my browser

### DISCUSSION THREAD

**Discussion by pityalvarez** [darkreader/darkreader#8903] [Ideas]
Isn't there a way to save the work done by the extension so it doesn't work slow next time?

**Comment by Gusted** [comment]
This isn't a actionable issue/discussion. Please be concrete.

**Reply by pityalvarez** [reply]
I use dynamic Dark mode. I close the tab. I open it again. It has to reload the whole process again that it takes several seconds. Is there a solution for this? So my experience is a little more snappier
